### PR TITLE
Merge changes from https://github.com/mksarge/patreon-js

### PR DESCRIPTION
**Pull request by 21echoes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "patreon",
   "version": "0.1.0",
```
Comment (derekr) [inline package.json]: We don't need to bump major (changes to the api) in the PR, but we shouldn't publish to NPM until we do.

File: readme.md [hunk 2]
```diff
@@ -73,18 +73,18 @@ const redirectURL = 'http://mypatreonapp.com/oauth/redirect'
...
-        const patreonAPIClient = patreonAPI(access_token)
-
-        patreonAPIClient(`/current_user`, (currentUserError, apiResponse) => {
-            if (currentUserError) {
-                console.error(currentUserError)
-                response.end(currentUserError)
-            }
-
-            response.end(apiResponse);
+    patreonOAuthClient.getTokens(oauthGrantCode, redirectURL)
```
Comment (derekr) [inline readme.md]: `getToken` w/ no s?

File: src/patreon.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import fetch from 'isomorphic-fetch'
 import { normalizeRequest, checkStatus, getJson } from './utils'
 
-function patreon(accessToken, config) {
```
Comment (derekr) [inline src/patreon.js]: Just a note: plan was to allow passing in custom hostname and all that to support requesting against a mock/non-prod environment.

File: readme.md (hunk 2 above)
Comment (21echoes) [reply readme.md]: yea i changed it back to `getTokens`, because it returns more than one token

File: src/patreon.js (hunk 3 above)
Comment (21echoes) [reply src/patreon.js]: sounds good, but i'd like to leave that out until it's actually used

**Pull request merged**